src/plugins/multimedia/multimedia-audio-fr.hbs

```diff
@@ -15,9 +15,9 @@
 <div class="row">
 	<section class="col-md-12">
 		<h2>Vue d'ensemble</h2>
-		<p>Pour l'audio seulement, le lecteur multimédia s'appuie sur la balise «&#160;<code>audio</code>&#160;» HTML5 natif et s'appuie sur Flash comme un mécanisme de secours lorsque la balise «&#160;<code>audio</code>&#160;» HTML5 n'est pas implémentée.</p>
+		<p>Pour l'audio seulement, le lecteur multimédia s'appuie sur la balise «&#160;<code>audio</code>&#160;» HTML5 natif.</p>
 		<p>Le scénario du lecteur multimédia repose sur l'élément «&#160;<code>progress</code>&#160;» HTML5 et utilise un polyfill lorsque l'élément n'est pas pris en charge.</p>
-		<p lang="en">The MP3 format (H264 codec) is the minimum requirement for the player because the Flash fallback relies on it and MP3 is a widely supported format on many mobile devices. An optional but highly recommended format, OGG Vorbis, should be added as well to allow some browsers such as Firefox that do not include native support for MP3 to leverage the HTML5 native performance gains.</p>
+		<p>Le format MP3 est le minimum requis pour la vidéo parce que c'est le format vidéo standard sur de nombreux navigateurs. Un OGG Vorbis facultative mais fortement recommandée devrait être ajouté aussi bien pour permettre à certains navigateurs tels que Firefox qui ne comprennent pas un support natif pour MP3 de tirer parti des gains de performance HTML5 indigènes.</p>
```

Comment:
> des gains de performance HTML5 indigènes

"indigènes" are not the right word to use in this context, it's means "indigenous people".

May be use instead: "des gains de performance de leur implémentations d'HTML5"

In fact the full sentence needs to be re-written in French but at least apply my suggestion it will be less bad.